rpms/generate_rpm_package.sh

```diff
@@ -45,14 +47,20 @@ clean() {
     exit ${exit_code}
 }
 
+ctrl_c() {
+    clean 1
+}
+
 build_rpm() {
     CONTAINER_NAME="$1"
     DOCKERFILE_PATH="$2"
 
+
     SOURCES_DIRECTORY="${CURRENT_PATH}/repository"
 
+
```

Comment:
This line is unnecessary.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/9fdbf7e2f0271d39ecdb594e355a87b08d902054.